frontend/src/container/ReduxTlmDispatcher.jsx

```diff
@@ -75,16 +74,11 @@ export class ReduxTlmDispatcher extends React.Component {
     this.props.dispatch(removeWorkspace(data.workspace))
   }
 
-  handleUserModified = data => {
-    this.props.dispatch(updateUser(data.user))
-  }
```

Comment:
why remove that handler?

Reply:
it was duplicated in the file, the result of merge I guess.
Both had the exact same implementation